source/reference/mongosync/mongosync-behavior.txt

```diff
@@ -179,6 +179,15 @@ primary shard to each database by means of a round-robin.
    restart the migration from the start. For more information, see
    :ref:`c2c-sharded-limitations`. 
 
+Config Shard Cluster
+''''''''''''''''''''
+
+Starting in 8.0, MongoDB introduces support for :ref:`config-shard-concept`.
```

Comment:
we can say "config shard clusters (also known as embedded config server clusters)" here, and then refer to embedded config server clusters below.